plugins/org.jkiss.dbeaver.ext.altibase/src/org/jkiss/dbeaver/ext/altibase/AltibaseUtils.java

```diff
@@ -16,13 +16,97 @@
  */
 package org.jkiss.dbeaver.ext.altibase;
 
+import org.jkiss.dbeaver.DBException;
+import org.jkiss.dbeaver.Log;
+import org.jkiss.dbeaver.model.exec.DBCException;
+import org.jkiss.dbeaver.model.exec.jdbc.JDBCPreparedStatement;
+import org.jkiss.dbeaver.model.exec.jdbc.JDBCResultSet;
+import org.jkiss.dbeaver.model.exec.jdbc.JDBCSession;
+import org.jkiss.dbeaver.model.impl.jdbc.JDBCUtils;
+import org.jkiss.dbeaver.model.runtime.DBRProgressMonitor;
+import org.jkiss.dbeaver.model.struct.DBSObject;
+import org.jkiss.dbeaver.model.struct.DBSObjectLazy;
+import org.jkiss.dbeaver.model.struct.cache.DBSObjectCache;
 import org.jkiss.utils.CommonUtils;
 
+import java.sql.SQLException;
+
 /**
  * Altibase utils
  */
 public class AltibaseUtils {
 
+    static final Log log = Log.getLog(AltibaseUtils.class);
+    
+    public static <PARENT extends DBSObject> Object resolveLazyReference(
+            DBRProgressMonitor monitor,
+            PARENT parent,
+            DBSObjectCache<PARENT, ?> cache,
+            DBSObjectLazy<?> referrer,
+            Object propertyId) throws DBException {
```

Comment:
Please use 4-spaces indentation everywhere. Usually, we format it this way
```suggestion
    public static <PARENT extends DBSObject> Object resolveLazyReference(
        DBRProgressMonitor monitor,
        PARENT parent,
        DBSObjectCache<PARENT, ?> cache,
        DBSObjectLazy<?> referrer,
        Object propertyId
    ) throws DBException {
```

Reply:
Hello, @E1izabeth 

I have updated the code as per your recommendations. However, I typically use a 4-spaces-only format policy in Eclipse, so there might still be some minor differences in indentation due to varying formatting policies.

If you could share the Eclipse format file with me, it would help ensure consistency and prevent this issue from recurring. 

Thanks again for your review :)

![image](https://github.com/user-attachments/assets/8305e353-bdb7-4624-b024-76ec25991a47)

Reply:
All members of the team use IDEA for development. So, unfortunately, we don't have the code style configuration for Eclipse

Reply:
Thank you for the corrections